docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -86,67 +86,66 @@ In the event of a total active region loss, the following data will be lost:
+    - For OpenShift, [Submariner](https://docs.redhat.com/en/documentation/red_hat_advanced_cluster_management_for_kubernetes/2.11/html/networking/networking#submariner) is recommended for handling multi-cluster networking. Specific implementation guides are not yet available.
+- Maximum network round trip time (**RTT**) between regions should not exceed **100 ms**.
+- Required open ports between the two regions:
+  - **9200** for Elasticsearch (for cross-region data push by Zeebe).
+  - **26500** for communication to the Zeebe Gateway from clients/workers.
+  - **26501** and **26502** for communication between Zeebe brokers and Zeebe Gateway.
+
+### Zeebe Cluster Configuration
+
+Only a combination for Zeebe broker counts and replication factors is supported:
+
+- `clusterSize` must be a multiple of **2** and at least **4** to evenly distribute brokers across the two regions.
+- `replicationFactor` must be **4** to ensure even partition distribution across regions.
+- `partitionCount` is unrestricted but should be chosen based on workload requirements. See [understanding sizing and scalability behavior](../../../components/best-practices/architecture/sizing-your-environment.md#understanding-sizing-and-scalability-behavior).
+- For more details on partition distribution, see [documentation on partitions](../../../components/zeebe/technical-concepts/partitions.md).
+
+### Regional Failure Management
+
+Customers are responsible for detecting regional failures and executing the [operational procedure](./../../operational-guides/multi-region/dual-region-ops.md).
+
+### Camunda 8 dual-region limitations
+
+| **Aspect**                     | **Details**                                                                                                                                                                                                                                                                                                                                                                                                            |
+| :----------------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Installation methods           | <p><ul><li>For **kubernetes** we recommend using a dual-region Kubernetes setup with the [Camunda Helm chart](/self-managed/setup/install.md) installed in two Kubernetes clusters.</li><li>For **other platforms**, using alternative installation methods (for example, with docker-compose) is not supported.</li></ul></p>                                                                                         |
```

Comment:
Let's keep it as before (it's not covered, not unsupported)

Reply:
yep, exactly, remove "unsupported" in favor of "not documented"

Reply:
I mean in the current docs we explicitly state `not supported`.

Reply:
if we want to go to not covered, then the 8.5 docs would need to be adjusted as well since nothing changed between 8.5 and 8.6.

Reply:
For my memory refreshing, OpenSearch is not compatible with DR, or we did not try because of complexity (as it would not be installed with Helm chart). Basically, checking if there's roadblocks to make DR work with OpenSearch in the future?

Reply:
We're using 2 separate ES clusters for DR, so that aspect should not be a blocker to have 2 OS clusters to connect to.